hadoop-ozone/dist/src/test/shell/compose_testlib.bats

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+source ../../main/compose/testlib.sh
+@test "Find test recursive, only on one level" {
+  run find_tests
+  echo "x${output}x}"
+  [[ "$output" == "./test1/test.sh ./test2/test.sh ./test4/test.sh" ]]
+}
+
+source ../../main/compose/testlib.sh
```

Comment:
Can you please explain what's the purpose of repeatedly loading the library being tested?  In my experience it works fine with a single load, but maybe you want to cover some case I'm not aware of.

```suggestion
load ../../main/compose/testlib.sh
```

or

```suggestion
```

Reply:
Yes, I learned from you that `load` can be used, but it doesnt work for me:

```
bats compose_testlib.bats
bats: /home/elek/projects/ozone/hadoop-ozone/dist/src/test/shell/../../main/compose/testlib.sh.bash does not exist
```

Don't know why....

Reply:
Upgrade bats to 1.2.1.

https://github.com/bats-core/bats-core/pull/282
https://github.com/bats-core/bats-core/releases/tag/v1.2.1